src/core/fallcurve/internal/vt_fc_signature_compute.c

```diff
@@ -33,7 +33,9 @@ static VT_INT fc_signature_calculate_37index(VT_UINT* raw_signature, VT_UINT sam
         }
     }
 
-    return -1;
+    return abs_custom(
```

Comment:
This logic is confusing to me. I get before returning -1 meaning it didnt find a match, but what does it mean to return this value now?

Reply:
So the fc_signature_calculate_37index function was returning -1 if the voltage response doesn't reach 37 % of the starting value by the time all SAMPLE_LENGTH data points are collected. When line 105 didn't use this returned value (index37), if -1 was returned by this function it entailed that legal and valid but SLOWER fallcurves which would reach 37% of starting value AFTER SAMPLE_LENGTH, would give same falltime as that of a fallcurve which reached 37% AT SAMPLE_LENGTH. Due to this bug, slower but legal/valid fallcurves were being successfully matched with fallcurves with high rate of voltage drop  as their falltimes were identical